packages/web-components/src/global/variables.css

```diff
@@ -30,3 +30,9 @@
     sans-serif;
   --font-serif: 'Bitter';
 }
+
+va-process-list li :is(h2, h3, h4, h5) {
+  margin-top: 0;
+  clear: none;
+  padding-top: 0.15em;
+}
```

Comment:
I'm not totally sold on this, and if we go with it then I'll make some additional changes to filenames and maybe have this imported separately. It's here to [replace these styles](https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/blob/8d2c965f4a3e60cd889649cc904828cb2d59fef8/packages/formation/sass/modules/_m-process-list.scss#L15-L19).

Web Component's can't use `::slotted()` to style nested elements for performance reasons, and there is [precedence for using utility classes inside the items in a process list](https://github.com/department-of-veterans-affairs/content-build/blob/d8111acf9910eb6d4a51186a33b046bdced44424/src/site/layouts/step_by_step.drupal.liquid#L69). We _could_ take the slotted nodes and rebuild them inside the shadow DOM so that we can style the headings directly (and this is what I originally did before [switching my approach](https://github.com/department-of-veterans-affairs/component-library/pull/265/commits/34c7488cb49ea16c95903a9c4988eca899348919)). If we did that then we would have to make certain utility classes available in the shadow DOM which adds complexity and just doesn't sound that great in general.

Reply:
@bkjohnson I think the approach you took is probably best because we do have that situation where utility classes live in a Process List and we need to make sure we can style those items. I kind of think doing utility classes in the shadow DOM would create more issues than it would solve. While this isn't the prettiest solution and slightly more verbose it probably grants us the greatest flexibility which is what will end up being requested by other VSP teams. I would also agree with you on having this live in a separate file, and I suspect we will run into this situation again with some of the other React Components we are converting to Web Components so this would provide us with a baseline to work against